src/pad/src/pad.tcl

```diff
@@ -336,14 +336,14 @@ sta::define_cmd_args "place_io_terminals" {inst_terms}
 proc place_io_terminals {args} {
   sta::parse_key_args "place_bondpad" args \
```

Comment:
I didn't get around to it, but the `sta::parse_key_args` has the wrong function name in the argument, could you correct that as well?

Reply:
Thanks for spotting that